app/default-config.js

```diff
@@ -0,0 +1,26 @@
+export const defaultConfig = {
+  commands: {
+    label: {
+      permission: "author-or-member",
+      allowed_labels: [],
+      enabled: true,
+    },
+    "remove-label": {
```

Comment:
```suggestion
    removeLabel: {
```

camel casing is very common in javascript.

do not look at release drafter as a good example 😆